src/app/child-dev-project/notes/model/note.ts

```diff
@@ -56,13 +64,21 @@ export class Note extends Entity {
   @DatabaseField({ innerDataType: "schema-embed", ext: EventAttendance })
   private childrenAttendance: Map<string, EventAttendance> = new Map();
 
-  @DatabaseField() date: Date;
-  @DatabaseField() subject: string = "";
-  @DatabaseField() text: string = "";
+  @DatabaseField({ label: "Date" }) date: Date;
+  @DatabaseField({ label: "Subject" }) subject: string = "";
+  @DatabaseField({ label: "Notes", editComponent: "EditLongText" })
+  text: string = "";
   /** IDs of users that authored this note */
-  @DatabaseField() authors: string[] = [];
+  @DatabaseField({
+    label: "SW",
+    viewComponent: "DisplayEntityArray",
```

Comment:
Similar question to the comment on the boolean value. Currently there is no datatype because no transformation is necessary. We could add the datatype but then it would be mostly for the fact of defining the view and edit components.